app/channel.go

```diff
@@ -1185,7 +1185,7 @@ func (a *App) ViewChannel(view *model.ChannelView, userId string, clearPushNotif
 		}
 	}
 
-	times := map[string]int64{}
+	var times map[string]int64
```

Comment:
Yeah, there's a difference. The old line created an empty map. This line creates a nil map.

There was no error before, but we shouldn't allocate things that are never used.